resolwe_bio/processes/support_processors/gff_to_gtf.yml

```diff
@@ -0,0 +1,36 @@
+# ============
+# Gff to gtf
+# ============
+
+---
+
+- slug: gff-to-gtf
+  name: Annotation (GFF3)
```

Comment:
Name of the processor is not descriptive enough. Use something like "Convert GFF to GTF"